tests/SimplifyTest.elm

```diff
@@ -5172,6 +5174,128 @@ a = str |> String.left n
         ]
 
 
+stringLeftTests : Test
+stringLeftTests =
+    describe "String.left"
+        [ test "should not report String.left that contains variables or expressions" <|
+            \() ->
+                """module A exposing (..)
+a = String.left b c
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectNoErrors
+        , test """should replace String.left 0 by \"\"""" <|
```

Comment:
I think the triple quotes in the test titles are not helping, since you're escaping the quotes inside the title anyway :sweat_smile:

Reply:
right. Those came from copying concat/join/repeat tests that had the same inconsistency. Changed those, too